service/src/test/java/bio/terra/drshub/util/SignedUrlTestUtils.java

```diff
@@ -95,7 +95,7 @@ public static void setupDrsResolutionServiceMocks(
             any(BearerToken.class),
             eq(true),
             nullable(String.class),
-            nullable(String.class));
+            eq(googleProject));
```

Comment:
I'm like 90% sure this is causing the tests to fail

Reply:
It totes was.